src/Dockerfile

```diff
@@ -112,6 +112,11 @@ ENV BASH_ENV=${CONTAINER_SCRIPTS_PATH}/scl_enable \
 RUN usermod -a -G root postgres && \
     /usr/libexec/fix-permissions --read-only "$APP_DATA"
 
+# fix incorrect permissions copied from host
```

Comment:
Some more explanation why this is needed (what issue was caused when not present) could be added either here as a comment, or inside the commit.